cli/events-plugin.md

```diff
@@ -0,0 +1,328 @@
+
+    ![](../img/cli-workspace-list.png)
+
+    To select a workspace, use ***aio console workspace select <workspace_id>***.
+
+    ![](../img/cli-workspace-select.png)
+
+## Installing and using the plugin
+
+Use the following syntax to install the Events Plugin for the Adobe I/O CLI and issue commands.
+
+```shell
+$ npm install -g @adobe/aio-cli-plugin-events
+$ aio COMMAND
+running command...
+$ aio (-v|--version|version)
+@adobe/aio-cli-plugin-events/1.0.0 darwin-x64 node-v10.18.1
+$ aio [COMMAND] --help
+USAGE
+  $ aio COMMAND
+...
+```
+
+## Commands
+
+There are a number of commands available when using the Events Plugin, please select an area from the following list to learn more: 
+
+* [Common command](#common-command)
+* [Registration commands](#registration-commands)
+* [Provider commands](#provider-commands)
+* [Event Metadata commands](#event-metadata-commands)
+
+### Common command
+
+```shell
+aio event
+```
+
+### Registration commands
+
+* [Manage your Adobe I/O Events Registrations](#manage-your-adobe-io-events-registrations)
+* [Get an Event Registration in your workspace](#get-an-event-registration-in-your-workspace)
+* [List Event Registrations in your workspace](#list-event-registrations-in-your-workspace)
+* [Create an Event Registration](#create-an-event-registration)
+
+#### Manage your Adobe I/O Events Registrations
+
+```shell
+USAGE
+$ aio event:registration
+ 
+OPTIONS
+-v, --verbose Verbose output
+--help Show help
+--version Show version
+```
+
+#### Get an Event Registration in your workspace
+
+```shell
+USAGE
+  $ ./bin/run event:registration:get REGISTRATIONID
+ 
+ARGUMENTS
+  REGISTRATIONID  Id of the registration to get
+ 
+OPTIONS
+  -j, --json     Output json
+  -v, --verbose  Verbose output
+  -y, --yml      Output yml
+  --help         Show help
+  --version      Show version
+ 
+ALIASES
+  $ ./bin/run console:reg:get
+```
+
+#### List Event Registrations in your workspace
+
+```shell
+USAGE
+$ ./bin/run event:registration:list
+ 
+OPTIONS
+-j, --json Output json
+-v, --verbose Verbose output
+-y, --yml Output yml
+--help Show help
+--version Show version
+ 
+ALIASES
+$ ./bin/run console:registration:ls
+$ ./bin/run console:registration:list
+$ ./bin/run console:reg:ls
+```
+
+#### Create an Event Registration
+
+```shell
+USAGE
+  $ ./bin/run event:registration:create
+ 
+OPTIONS
+  -j, --json     Output json
+  -v, --verbose  Verbose output
+  -y, --yml      Output yml
+  --help         Show help
+  --version      Show version
+```
+
+### Provider Commands
+
+* [Manage your Adobe I/O Events Providers](#manage-your-adobe-io-events-providers)
+* [Get details for a Provider by its ID](#get-details-for-a-provider-by-its-id)
+* [List all Providers for the Organization](#list-all-providers-for-the-organization)
+* [Create a new Provider](#create-a-new-provider)
+* [Update an existing Provider](#update-an-existing-provider)
+
```

Comment:
We also have a DELETE provider https://github.com/adobe/aio-cli-plugin-events#binrun-eventproviderdelete-providerid

NOTE: The Readme will be updated soon to have aio instead of ./bin/run as well